routes/index.js

```diff
@@ -38,7 +38,9 @@ router.get('/events/:appId/exec', eventLogger.appExecEvents);
 router.get('/apps', appsController.list);
 router.get('/apps/new', appsController.new);
 router.get('/apps/:appId', appsController.show);
+router.get('/apps/update/:appId', appsController.showUpdatePage);
 router.post('/apps', upload.single('file'), appsController.create);
+router.post('/apps/update/:appId', upload.single('file'), appsController.update);
```

Comment:
I think the convention here would be `/apps/:appId`. Again, not really that important

Reply:
So, by my understanding the convention would be `PUT /apps/:appId`.
However, since we're working with web forms, we're limited to GET and POST.

So, maybe `POST /apps/:appId/edit` would be a reasonable compromise?